install_base_ubuntu.sh

```diff
@@ -59,13 +59,20 @@ install_nodejs() {
     apt-get install -y nodejs npm
 }
 
+install_doc_extras() {
```

Comment:
I hid the extra deps behind this. Dunno if it would make sense it install it unconditionally.

Reply:
I would keep it optional... as per the SDK in the CI loop I don't think we need those bits and thus we can save a bit of download/installation  time, isn't it?

Reply:
Okay, I'll try to tweak the doc build to make it obvious that you need those extra bits when plantuml fails